burnman/tools.py

```diff
@@ -643,3 +644,109 @@ def check_eos_consistency(m, P=1.e9, T=300., tol=0.01, verbose=False):
             print('Not satisfied all EoS consistency constraints for {0:s}'.format(m.to_string()))
             
     return consistency
+
+
+def _pad_ndarray_inverse_mirror(array, padding):
```

Comment:
True, that's how gaussian_filter works, but in order to add a new "mode" (which is necessary for our data) I would have to write my own version of gaussian_filter and use gaussian1d. I felt that my current strategy (using scipy's function rather than rewriting it) was slightly more sensible, but I can certainly rewrite gaussian_filter if you want? Processing would certainly be faster.